Gems/Atom/RPI/Code/Include/Atom/RPI.Public/WindowContext.h

```diff
@@ -37,6 +37,15 @@ namespace AZ
             WindowContext() = default;
             ~WindowContext() = default;
 
+            //! Enum to describe swapchain type
+            enum class SwapChainMode : uint32_t
+            {
+                XrLeft = 0,
+                XrRight,
+                Default,
+                Count
+            };
```

Comment:
suggestion: Since there is a `Default` value in the enum, maybe that should be the first entry with value 0. That way it'd be safer for init values of 0 to translate to the default value.